front-end/src/ui-components/Button.tsx

```diff
@@ -65,7 +78,7 @@ export default styled(Button)`
 			&:hover, &:focus {
 				background-color: red_secondary;
 				color: white;
-				box-shadow: 0 0 0 1px #A6ACB3 inset !important;
+				box-shadow: 0 0 0 1px #D94C3D inset;
```

Comment:
please use css variable for color

Reply:
thanks - replaced with box-shadow vars (box-shadow doesn't recognise the color variables)